### ISSUE TITLE
[BUG] Response status code does not indicate success:  403 during Visual Studio 2022 17.10 startup

### ISSUE THREAD

**Issue by MagicAndre1981** [luminous-software/start-page-plus-64-bit#42]
**Describe the bug**

During Startup of VS2022 I get an error message from StartIsBack+ telling that something went wrong.

**To Reproduce**
1. start Visual Studio
2. See error

**Expected behavior**

**Screenshots**

![image](https://github.com/luminous-software/start-page-plus-64-bit/assets/8274816/fb5d49b3-d906-4da8-814f-17fb846a1a20)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 11 23H2
- VS2022 17.10

**Additional context**

**Comment by yannduran** [comment]
Hi André

SP+ is simply reporting an error that it received from a web request. The error itself had to have come from amazonaws.com

The only place in SP+ that I would ever expect to see such an error is when opening a Developer News item, which I know you don’t use.

Does the error keep happening? Or is it just a one-off? What are you doing just before you see the error?

Yann

**Comment by MagicAndre1981** [comment]
yes, it comes from the url https://vsstartpage.blob.core.windows.net/news/vs . Even when I turn Developer News off the extension still tries to read the URL.

Btw, can you please merge my PR #37 ?

**Comment by ruslan-mogilevskiy** [comment]
Same error here in vs2022 with disabled 'developer news'. The error appears 1-2 times once I open Start+ in VS for the first time.

**Comment by MagicAndre1981** [comment]
I use a [different feed](https://[visualstudiomagazine.com/RSS-Feeds/Blogs.aspx](https://visualstudiomagazine.com/RSS-Feeds/Blogs.aspx)) now to avoid this error
